locale/bh.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$meta": {
+        "label": "Bosnian",
+        "humanizer_language": "bh"
+    },
+    "restarter": {
+        "start_timeout": "Server nije uspio započeti na vrijeme",
+        "crash_detected": "Server se zaustavio",
+        "partial_crash_warn": "Zbog djelomičnog pada, Server će se ponovo pokrenuti za 1 minutu. Isključiti te se sada.",
+        "partial_crash_warn_discord": "Zbog djelomičnog pada, **%{servername}** će se restartovati",
+        "schedule_reason": "restart je zakazan za %{time}",
+        "schedule_warn": "Ovaj server ce se restartovati za %{smart_count} minute. Molim vas se isključitite. |||| Ovaj server ce se restartovati za %{smart_count} minute.",
+        "schedule_warn_discord": "**%{servername}** restarta se za %{smart_count} minuta. |||| **%{servername}** restarta se za %{smart_count} minuta."
```

Comment:
is the plural vs singular correct in here? have you tested it in-game?

Reply:
yes they are and it isnt tested